samples/geotag.cpp

```diff
@@ -794,7 +794,7 @@ int main(int argc,const char* argv[])
     shorts["-v"] = "-verbose";
     shorts["-V"] = "-version";
     shorts["-d"] = "-dst";
-    shorts["-a"] = "-adjust";
+    shorts["-j"] = "-adjust";
```

Comment:
Does this need a change in the documentation? It changes the way this program behaves.

Reply:
I do not know if there is documentation for this app. If you know there is, please tell me where I can find it and I will update it.

Note that it is a sample application, so it should not affect much.

Reply:
Yes, it is in the manpages: http://exiv2.org/manpage.html